plugin/src/main/java/dev/regadas/trino/pubsub/listener/PubSubEventListener.java

```diff
@@ -94,7 +93,12 @@ public void onSuccess(String id) {
                         }
 
                         public void onFailure(Throwable t) {
-                            LOG.log(Level.SEVERE, "Failed to publish event", t);
+                            LOG.log(
+                                    Level.SEVERE,
+                                    "Failed to publish event, error: "
+                                            + t.getMessage()
+                                            + " , event: "
+                                            + message.getData().toStringUtf8());
```

Comment:
a bit on the fence here ... `message` will be **very large** it will probably be split in multiple entries (i.e Cloud Monitoring). Regarding `t.getMessage()` by attaching the `Throwable` we already get it.

One thing though is that we could log

```java
var msg = "Failed to publish {"+ eventType +"} event for query {"+ queryId +"}"
LOG.log(Level.SEVERE, msg, t);
```

It would really be better if we find a way to log only the error and that be very explicit.

Reply:
but I guess that's on pubsub side to actually give a really good message.